### DISCUSSION TITLE
Restricting WordPress entry points

### DISCUSSION THREAD

**Discussion by szepeviktor** [serversideup/docker-php#69] [Ideas & Feature Requests]
Seeing https://github.com/serversideup/docker-php/blob/4742cbc3dfe8189a04aea27801c800deb5ee2ae7/php/8.0/fpm-apache/etc/apache2/sites-available/000-default.conf#L59-L64

You may be interested in https://github.com/szepeviktor/debian-server-tools/blob/master/webserver/apache-conf-available/wordpress.inc.conf

**Comment by szepeviktor** [comment]
Maybe my solution is concentrating on denying requests.
Listing all allowed URL paths may be better!

**Comment by jaydrogers** [comment]
Just adding a note from @szepeviktor here: https://github.com/serversideup/docker-php/commit/ddc5e5ae0d74031ef6232ea02b33ea93ef9c4f8d#commitcomment-53836815

Looking at this for the next release 👍
